search_by_car.py

```diff
@@ -0,0 +1,61 @@
+"""Take in a command line argument in the form:
+python search_by_car.py <db_filename>.sqlite <car_name>
```

Comment:
I would replace the reading and parsing of command line arguments with a raw_input() function similar to my get_input() function in the convert_from_user module from Milestone 5.